### PR TITLE
Add signing manifest and script to update it with production values

### PR DESCRIPTION
**Pull request by TravisEz13**

### CODE REVIEW

File: tools/releaseBuild/signing.xml [hunk 1]
```diff
@@ -0,0 +1,49 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<SignConfigXML>
+  <!-- ****Begin**** Should be BothDual - Dual (Sha256 and Sha1) AuthenticodeDual) and StrongName ******** -->
```
Comment (daxian-dbw) [inline tools/releaseBuild/signing.xml]: "StorngName" is gone

File: tools/releaseBuild/signing.xml [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+    <file src="__INPATHROOT__\Microsoft.PowerShell.SDK.dll" signType="BothDual" dest="__OUTPATHROOT__\Microsoft.PowerShell.SDK.dll" />
+-->
+
+ <!-- ****Begin**** AuthenticodeDual - Dual (Sha256 and Sha1) Authenticode ************* -->
+
+    <file src="__INPATHROOT__\pwsh.exe" signType="AuthenticodeDual" dest="__OUTPATHROOT__\pwsh.exe" />
+
+
+ <!--
+    <file src="__INPATHROOT__\Install-PowerShellRemoting.ps1" signType="AuthenticodeDual" dest="__OUTPATHROOT__\Install-PowerShellRemoting.ps1" />
```
Comment (daxian-dbw) [inline tools/releaseBuild/signing.xml]: This script is not in the parent folder of publish. File an issue.

File: tools/releaseBuild/signing.xml [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+  -->
+ <!-- ****Begin**** Authenticode - Authenticode SHA256 ************* -->
```
Comment (daxian-dbw) [inline tools/releaseBuild/signing.xml]: Add a comment why those files cannot be dual signed (they are scripts)

File: tools/releaseBuild/signing.xml (hunk 2 above)
Comment (daxian-dbw) [reply tools/releaseBuild/signing.xml]: Another ps1 file is coming for etw provider.

File: tools/releaseBuild/updateSigning.ps1 [hunk 4]
```diff
@@ -0,0 +1,31 @@
+# Script for use in VSTS to update signing.xml
+
+# Parse the signing xml
+$signingXmlPath = Join-Path -Path $PSScriptRoot  -ChildPath 'signing.xml'
+$signingXml = [xml](Get-Content $signingXmlPath)
+
+# Get any variables to updating 'signType' in the XML
+# Define a varabile named `<signTypeInXml>SignType' in VSTS to updating that signing type
```
Comment (daxian-dbw) [inline tools/releaseBuild/updateSigning.ps1]: Add a concrete example.

**Review by daxian-dbw — APPROVED** [review]
LGTM. Some comments left.

File: tools/releaseBuild/signing.xml (hunk 2 above)
Comment (TravisEz13) [reply tools/releaseBuild/signing.xml]: https://github.com/PowerShell/PowerShell/issues/5398 filed

**Review by daxian-dbw — APPROVED** [review]

**Review by adityapatwardhan — APPROVED** [review]

**Pull request merged**